src/containers/other/NFTView/NFTView.types.ts

```diff
@@ -0,0 +1,12 @@
+import {
+	DomainTransferEvent,
+	DomainMintEvent,
+	DomainBidEvent,
+	DomainSaleEvent,
+} from '@zero-tech/zns-sdk/lib/types';
+
+export type DomainEvents =
+	| DomainTransferEvent
+	| DomainMintEvent
+	| DomainBidEvent
+	| DomainSaleEvent;
```

Comment:
Nice union types! Btw there is not this type defined in SDK? @cc @colbr

Reply:
I don't think so! At least not in the current version